Deprecated/stop_all_C7.py

```diff
@@ -9,7 +9,7 @@
 
 from __future__ import absolute_import, division, print_function
 
-from hera_mc import mc, cm_utils, part_connect, cm_handling, geo_handling, cm_hookup, cm_part_revisions
+from hera_mc import mc, cm_utils, part_connect, cm_handling, cm_hookup, cm_part_revisions
```

Comment:
Yeah, I fixed this when it was not in deprecated, but when I rebased on master it had been moved. I can remove it if really desired, but it's importing something that's not used, so it doesn't really change anything.